java/src/jmri/jmrit/roster/FullBackupImportAction.java

```diff
@@ -97,13 +97,13 @@ public void close() {
             acceptAllDup = false;  // skip prompting for dups and accept all
             
             while ((entry = zipper.getNextEntry()) != null) {
-                log.debug(String.format("Entry: %s len %d (%d) added %TD content: %s",
+                log.debug("Entry: {} len {} ({}) added {} content: {}",
```

Comment:
I don't think this is a correct change. The `%TD` seems to me as a date formatted as `"%tm/%td/%ty"`, but I'm not entirely sure. See https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html#syntax.

Reply:
You can use `String.format("%TD", value)` as a parameter to log.debug() to solve this.

Reply:
Hi @danielb987 

The mm/dd/yy format doesn't work too well for me as native format is dd/mm/yy .
I've tweaked to the ISO 8601 yyyy-mm-dd hh:mm:ss format for improved understandability.